### ISSUE TITLE
Allow LABELS as placeholder in Dynatrace.sli

### ISSUE THREAD

**Issue by grabnerandi** [keptn-contrib/dynatrace-sli-service#34]
Right now the Dynatrace-SLI Service only allows 4 placeholders: $PROJECT, $STAGE, $SERVICE, $DEPLOYMENT.
Many discussions we have around using Keptn for Quality Gates only or for "Performance as a Self-Service" have shown that these 4 placeholders are not enough. 

To give you a concrete example from Discovery:
They are using Jenkins to deploy their service and then put a handful of custom tags on them. They also use Jenkins and leverage the x-dynatrace header to pass in things like TSN (Test Step Name: homepage, search, checkout) and LTN (Load Test Name: loadtest_job1, loadtest_job2, ...). Especially LTN is dynamic and changes with every Jenkins pipeline execution as the Jenkins Job ID is used in that Test NAme.

In order for Discovery to be able to use SLIs they would need to be able to reference these "dynamic values" - otherwise they would need to auto-generate the SLI for every Jenkins Job, make their own replacements, upload it to the keptn git repo and run the evaluation.

In order to solve this I suggest we take the labels that are passed to the Deployment-Finished, Test-Finished or Start-Evaluation event - make sure these labels are passed to the SLI Provider, e.g: Dynatrace SLI Provider and then use them in the SLIs as placeholder.
Here is an example:
```
contenttype": "application/json",
  "data": {
    "labels": {
      "testname" : "mytest_1",
      "servicetag1" : "mytag",
      "servicetag1" : "mytag"
    },
    "project": "simpleproject",
    "service": "simplenode",
    "stage": "staging",
    "start": "2019-11-21T11:00:00.000Z",
    "end": "2019-11-21T11:05:00.000Z",
    "teststrategy": "performance"
  },
  "source": "jenkins",
  "type": "sh.keptn.event.start-evaluation",
}
```

The Dynatrace SLI could then look like this
```
spec_version: '1.0'
indicators:
  throughput: "builtin:service.requestCount.total:merge(0):sum?scope=tag($LABEL.servicetag1),tag($LABEL.servicetag2)"
  svccalls_test_invoke: "calc:service.teststepservicecalls:filter(eq(Test Name,$LABEL.testname)):merge(0):avg?scope=tag($LABEL.servicetag1),tag($LABEL.servicetag2)"
``` 

This would give our users the flexiblity to pass in any custom meta data via labels to Keptn and in the end to the Dynatrace SLI Provider.

**Comment by christian-kreuzberger-dtx** [comment]
@grabnerandi has this been implemented in PR #40? If so, please close it!

**Comment by grabnerandi** [comment]
Implemented in #40
